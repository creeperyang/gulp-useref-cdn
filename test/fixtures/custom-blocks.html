<html>
<head></head>
<body>
<!-- build:custom customResult -->
someContent
<!-- endbuild -->
</body>
</html>
