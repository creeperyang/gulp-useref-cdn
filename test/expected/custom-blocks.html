<html>
<head></head>
<body>
customResult
</body>
</html>
